### PR TITLE
Move Prometheus custom metrics and alerting tutorials

### PR DESCRIPTION
**Pull request by chrkl**
**Description**

Changes proposed in this pull request:

- Move Prometheus custom metrics and alerting tutorials under the custom Prometheus tutorial
- Align instructions to custom Prometheus

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/15914

### CODE REVIEW

**Review by lindnerby — DISMISSED** [review]

File: prometheus/monitoring-alert-rules/README.md [hunk 1]
```diff
@@ -23,7 +22,7 @@ You need access to the `kyma-system` Namespace to execute the described steps.
 2. Run the `port-forward` command on the `monitoring-prometheus` service to access the Prometheus dashboard.
 
     ```bash
-    kubectl port-forward -n kyma-system svc/monitoring-prometheus 9090:9090
+    kubectl -n ${KYMA_NS} port-forward $(kubectl -n ${KYMA_NS} get service -l app=kube-prometheus-stack-prometheus -oname) 9090
```
Comment (Teneroy) [inline prometheus/monitoring-alert-rules/README.md:25]: Maybe it should be KYMA_EXAMPLE_NS in the context of this tutorial?

File: prometheus/monitoring-alert-rules/README.md [hunk 2]
```diff
@@ -7,11 +7,10 @@ This example shows how to deploy and view alerting rules in Kyma.
...
 * Kyma as the target deployment environment.
+* Deployed custom kube-prometheus-stack as described in the [prometheus](../) example.
 
 ## Installation
 
-You need access to the `kyma-system` Namespace to execute the described steps.
-
 ### Add a new alerting rule
 
 1. Create the PrometheusRule resource holding the configuration of your alerting rule.
```
Comment (Teneroy) [inline prometheus/monitoring-alert-rules/README.md:16]: Maybe we should put there a path to the raw version of the file as we did it in prometheus tutorial?

File: prometheus/monitoring-custom-metrics/README.md [hunk 3]
```diff
@@ -44,7 +45,7 @@ This example shows how to expose custom metrics to Prometheus with a Golang serv
 1. Run the `port-forward` command on the `monitoring-prometheus` service:
 
     ```bash
-    kubectl port-forward -n kyma-system svc/monitoring-prometheus 9090:9090
+    kubectl -n ${KYMA_NS} port-forward $(kubectl -n ${KYMA_NS} get service -l app=kube-prometheus-stack-prometheus -oname) 9090
```
Comment (Teneroy) [inline prometheus/monitoring-custom-metrics/README.md:48]: Maybe it should be KYMA_EXAMPLE_NS in the context of this tutorial?

File: prometheus/monitoring-grafana-dashboard/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+* Deployed custom kube-prometheus-stack as described in the [prometheus](../) example.
+* Deployed [custom-metrics example](../monitoring-custom-metrics).
+
+## Create the dashboard
+
+1. [Access Grafana](../README.md#verify-the-installation).
+2. Add a new dashboard with a new panel.
+3. For your new query, select **Prometheus** from the data source selector.
+4. Pick the `cpu_temperature_celsius` metric.
+5. To retrieve the latest metric value on demand, activate the **Instant** switch.
```
Comment (Teneroy) [inline prometheus/monitoring-grafana-dashboard/README.md:17]: Would be really nice to specify a bit where it is on Grafana dashboard. I had a really hard time understanding what the Instant switch actually means

File: prometheus/monitoring-grafana-dashboard/README.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+6. From the visualization panels, select the **Gauge** dashboard type.
+7. Save your changes and provide a name for the dashboard.
+
+## Configure the dashboard
+
+1. To edit the dashboard settings, go to the **Panel Title** options and select **Edit**.
+2. Find the **Field** tab and set the measuring unit to Celsius degrees, indicating the metric data type.
```
Comment (Teneroy) [inline prometheus/monitoring-grafana-dashboard/README.md:24]: Would be nice to mention that we choose this value in Unit section

File: prometheus/monitoring-grafana-dashboard/README.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
+# Create a Grafana dashboard
+
+Follow these sections to create the Gauge dashboard type for the `cpu_temperature_celsius` metric.
```
Comment (IwonaLanger) [inline prometheus/monitoring-grafana-dashboard/README.md:3]:
```suggestion
Follow these instructions to create the Gauge dashboard type for the `cpu_temperature_celsius` metric.
```

File: prometheus/monitoring-grafana-dashboard/README.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+4. Pick the `cpu_temperature_celsius` metric.
+5. To retrieve the latest metric value on demand, activate the **Instant** switch.
+6. From the visualization panels, select the **Gauge** dashboard type.
+7. Save your changes and provide a name for the dashboard.
+
+## Configure the dashboard
+
+1. To edit the dashboard settings, go to the **Panel Title** options and select **Edit**.
+2. Find the **Field** tab and set the measuring unit to Celsius degrees, indicating the metric data type.
+3. Set the minimum metric value to `60` and the maximum value to `90`, indicating the `cpu_temperature_celsius` metric value range.
```
Comment (IwonaLanger) [inline prometheus/monitoring-grafana-dashboard/README.md:25]:
```suggestion
3. To indicate the `cpu_temperature_celsius` metric value range, set the minimum metric value to `60` and the maximum value to `90`. 
```

File: prometheus/monitoring-grafana-dashboard/README.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+4. For the dashboard to turn red once the CPU temperature reaches and exceeds 75°C, set a red color threshold to `75`.
+5. Go to the **Panel** tab and title the dashboard, for example, `CPU Temperature`.
+6. To display this range on the dashboard, make sure that under **Panel > Display**, the threshold labels and threshold markers are activated.
+7. Save your changes. We recommend that you add a note to describe the changes made.
```
Comment (IwonaLanger) [inline prometheus/monitoring-grafana-dashboard/README.md:29]:
```suggestion
7. Save your changes. It is recommended that you add a note to describe the changes made.
```

File: prometheus/monitoring-grafana-dashboard/README.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+## Check the dashboard
+
+Refresh the browser to see how the dashboard changes according to the current value of the `cpu_temperature_celsius` metric.
+
+- If the current metric value ranges from 60 to 74 degrees Celsius, it turns **green**.
+- If the current metric value ranges from 75 to 90 degrees Celsius, it turns **red**.
+
+## Add the dashboard as Kubernetes resource
+
+When you create a dashboard to monitor one of your applications (Function, microservice,...), we recommend that you define the dashboard as a Kubernetes ConfigMap resource. In this case, a Grafana sidecar automatically loads the Dashboard on Grafana startup. Following that approach, you can easily keep the dashboard definition together with the Kubernetes resource definitions of your application and port it to different clusters.
```
Comment (IwonaLanger) [inline prometheus/monitoring-grafana-dashboard/README.md:40]:
```suggestion
When you create a dashboard to monitor one of your applications (Function, microservice,...), it is recommended that you define the dashboard as a Kubernetes ConfigMap resource. In this case, a Grafana sidecar automatically loads the Dashboard on Grafana startup. Following that approach, you can easily keep the dashboard definition together with the Kubernetes resource definitions of your application and port it to different clusters.
```

**Review by IwonaLanger — DISMISSED** [review]

**Comment by chrkl** [comment]
The governance job is unhappy with links to raw files that are created by this PR. Removed the links temporarily and will add them again in a second PR.

**Review by lindnerby — APPROVED** [review]

**Comment by chrkl** [comment]
@IwonaLanger could you review again please?

File: prometheus/README.md [hunk 10]
```diff
@@ -114,6 +114,14 @@ prometheus.io/path: /myMetrics # optional, configure the path under which the me
 
 You can try it out by removing the ServiceMonitor from the previous example and instead providing the annotations to the Service manifest.
 
+### Setup alerting rules
```
[the remark this answers is not in the document]
Comment (chrkl) [reply prometheus/README.md]: Right, "Set up" makes more sense.

**Review by IwonaLanger — APPROVED** [review]

**Review by lindnerby — APPROVED** [review]

**Pull request merged**